docs/api.md

```diff
@@ -98,13 +111,13 @@ POST to this URL to create a new session.
 Body:
 
     {
-      "username": <string>,
+      "email": <string>,
```

Comment:
This will fail - see https://github.com/Metaswitch/ellis/blob/cbc6d5683904233d6bc07090bd182ffb5a575475/src/metaswitch/ellis/api/session.py#L54.